app/register/page.tsx

```diff
@@ -83,36 +107,86 @@ const Register = () => {
             </h1>
             <p className="pb-4 font-normal leading-7 text-zinc-500">
               If you have an existing account{' '}
-              <Link href="/login" className="hover:text-orange-600">
-                Login!
-              </Link>
+              <LinkCustom href="/login">Login!</LinkCustom>
             </p>
-            <Input
-              type="email"
-              value={email}
-              placeholder="Email"
-              onChange={handleEmail}
-            />
-            <Input
-              type="password"
-              value={password}
-              placeholder="Password"
-              onChange={handlePassword}
-            />
-            <Input
-              type="password"
-              value={confirmPassword}
-              placeholder="Confirm Password"
-              onChange={handleConfirmPassword}
-            />
-            <Button
-              label="Register"
-              disabled={handleDisabled()}
-              onClick={handleRegister}
-            />
-            <Link href="/login" className="hover:text-orange-600">
-              Login to get started playing
-            </Link>
+
+            <Form {...form}>
+              <form onSubmit={form.handleSubmit(onSubmit)}>
```

Comment:
I reckon this should have the same styles as the one on the login page?

Reply:
Updated.